sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/HiveOrcQuerySuite.scala

```diff
@@ -208,4 +208,14 @@ class HiveOrcQuerySuite extends OrcQueryTest with TestHiveSingleton {
       }
     }
   }
+
+  test("SPARK-23340 Empty float/double array columns raise EOFException") {
```

Comment:
Just to confirm that these two tests are in the master branch, right?

Reply:
nvm. I found the original PR has them too. This is just a backport. Normally, we often refer to the original PR https://github.com/apache/spark/pull/20511 in the PR description